src/mcts/search.cc

```diff
@@ -790,16 +822,18 @@ void SearchWorker::RunNNComputation() {
 
 // 5. Populate computed nodes with results of the NN computation.
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-void SearchWorker::FetchNNResults() {
-  if (computation_->GetBatchSize() == 0) return;
-
-  // Copy NN results into nodes.
+void SearchWorker::FetchMinibatchResults() {
+  // Copy NN results, or cached/duplicate results, into nodes.
+  // The latter is much simpler.
```

Comment:
Describes why we have one loop with a shortcircuit `break`, as opposed to something fancier like two different loops or something

Reply:
To code reader (in this case, me), it's not clear that it describes that, so I'd remove that.
Instead, it may make sense to add one sentence comment in the code in lines 831-834 where it's actually happening.
Something like:
```cpp
  // For terminal nodes the V value is know without need to request it from NN.
```